src/Features/Core/Portable/InlineHints/AbstractInlineTypeHintsService.cs

```diff
@@ -76,15 +76,53 @@ internal abstract class AbstractInlineTypeHintsService : IInlineTypeHintsService
                     continue;
 
                 finalParts.AddRange(suffix);
+                var taggedText = finalParts.ToTaggedText();
+                var displayString = GetDisplayStringFromParts(taggedText);
+
+                TextChange? textChange = null;
+                if (insertSpan is not null)
+                {
+                    textChange = new TextChange(insertSpan.Value, displayString);
+                }
```

Comment:
```suggestion
                var textChange = insertSpan is null
                    ? null
                    : new TextChange(insertSpan.Value, displayString);
```

Reply:
need to explicitly cast null to TextChange?, otherwise it can't determine the type of the expression